00:00:00.013558 main     VirtualBox COM Server 6.1.28 r147628 win.amd64 (Oct 18 2021 18:49:03) release log
00:00:00.014167 main     Log opened 2022-02-28T13:28:28.280191900Z
00:00:00.014168 main     Build Type: release
00:00:00.014183 main     OS Product: Windows 11
00:00:00.014186 main     OS Release: 10.0.22000
00:00:00.014187 main     OS Service Pack: 
00:00:00.159710 main     DMI Product Name: Inspiron 15 3511
00:00:00.177883 main     DMI Product Version: 
00:00:00.177897 main     Firmware type: UEFI
00:00:00.178588 main     Secure Boot: VERR_PRIVILEGE_NOT_HELD
00:00:00.179721 main     Host RAM: 7930MB (7.7GB) total, 1803MB available
00:00:00.179729 main     Executable: C:\Program Files\Oracle\VirtualBox\VBoxSVC.exe
00:00:00.179730 main     Process ID: 11392
00:00:00.179731 main     Package type: WINDOWS_64BITS_GENERIC
00:00:00.433639          VirtualBox: object creation starts
00:00:00.435549          Home directory: 'C:\Users\reddy\.VirtualBox'
00:00:00.436361          Installed Drivers:
00:00:00.443953            C:\WINDOWS\system32\DRIVERS\VBoxNetLwf.sys (Version: 6.1.28.47628)
00:00:00.445897            C:\WINDOWS\system32\DRIVERS\VBoxUSBMon.sys (Version: 6.1.28.47628)
00:00:00.447574            C:\WINDOWS\system32\DRIVERS\VBoxDrv.sys (Version: 6.1.28.47628)
00:00:00.455387          Loading settings file "C:\Users\reddy\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:00.460606          usbLibGetDevices: Starting USB device enumeration
00:00:00.462620          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&281396e5&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 6
00:00:00.462676          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&281396e5&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
00:00:00.464119          usbLibGetDevices: Found 3 USB devices, 0 captured
00:00:00.465694          USB: Unknown USB device detected (idVendor: 0x0bda, idProduct: 0xc829)
00:00:00.466693          USB: Unknown USB device detected (idVendor: 0x0c45, idProduct: 0x6730)
00:00:00.667248          netIfIsWireless: CreateFile on '\\.\{F44AD32E-5013-4F96-BB07-135880781B10}' failed with rcWin=2 (0x2) - ignoring
00:00:00.690075          HostDnsMonitor: initializing
00:00:00.704582          HostDnsMonitor: updating information
00:00:00.705106          HostDnsMonitor: old information
00:00:00.705109            no server entries
00:00:00.705111            no domain set
00:00:00.705111            no search string entries
00:00:00.705112          HostDnsMonitor: new information
00:00:00.705113            server 1: 192.168.43.1
00:00:00.705115            no domain set
00:00:00.705115            no search string entries
00:00:00.882016          VD: VDInit finished with VINF_SUCCESS
00:00:00.891764          Loading settings file "C:\Users\reddy\VirtualBox VMs\kalilinux\kalilinux.vbox" with version "1.16-windows"
00:00:00.930581          OCI: Local config file 'C:\Users\reddy\.VirtualBox\oci_config' does not exist
00:00:00.930752          OCI: Original config file 'C:\Users\reddy\.oci\config' does not exist
00:00:00.930754          OCI: Reading profiles finished with status ERROR_SUCCESS
00:00:00.931256          ExtPack: Created cloud provider 'OCI' (hrc=ERROR_SUCCESS)
00:00:00.931337          VirtualBox: object created
00:00:03.084076 USBPROXY usbLibGetDevices: Starting USB device enumeration
00:00:03.089894 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&281396e5&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
00:00:03.090286 USBPROXY usbLibGetDevices: Found 3 USB devices, 0 captured
00:00:26.352814 main     VirtualBox: object deletion starts
00:00:26.352930 main     ERROR [COM]: aRC=VBOX_E_OBJECT_IN_USE (0x80bb000c) aIID={ad47ad09-787b-44ab-b343-a082a3f2dfb1} aComponent={MediumWrap} aText={Medium 'C:\Users\reddy\VirtualBox VMs\kalilinux\kalilinux.vdi' cannot be closed because it is still attached to 1 virtual machines}, preserve=false aResultDetail=0
00:00:26.358211 main     HostDnsMonitor: shutting down ...
00:00:26.359574 main     HostDnsMonitor: shut down
00:00:26.373978 Watcher  ERROR [COM]: aRC=E_ACCESSDENIED (0x80070005) aIID={d0a0163f-e254-4e5b-a1f2-011cf991c38d} aComponent={VirtualBoxWrap} aText={The object is not ready}, preserve=false aResultDetail=0
00:00:26.378841 main     VirtualBox: object deleted
